src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -121,8 +121,19 @@ private enum EXECTYPE
         // cached metadata
         private _SqlMetaDataSet _cachedMetaData;
 
-        private Dictionary<int, SqlTceCipherInfoEntry> keysToBeSentToEnclave;
-        private bool requiresEnclaveComputations = false;
+        internal Dictionary<int, SqlTceCipherInfoEntry> keysToBeSentToEnclave;
+        internal bool requiresEnclaveComputations = false;
+        private bool ShouldCacheEncryptionMetadata
```

Comment:
It would be a bit neater is this were moved down so it wasn't in the middle of the field declarations. The file is a bit of a mess though so if you don't want to do it in this PR that's no problem.

Reply:
It'll get cleaned up in the merge process eventually. Thanks for moving this one.